packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -2,10 +2,20 @@ import { UserValidationRule } from "./user-validation-rule.enum.js";
 
 const UserValidationMessage = {
 	EMAIL_REQUIRED: "Email is required.",
+	EMAIL_TOO_LONG: `Email must be less than ${String(UserValidationRule.EMAIL_MAXIMUM_LENGTH)} characters long.`,
 	EMAIL_WRONG: "Email is wrong.",
 	NAME_REQUIRED: "Name is required.",
 	NAME_TOO_LONG: `Name is too long (> ${String(UserValidationRule.NAME_MAXIMUM_LENGTH)} symbols).`,
```

Comment:
Let's also modify this error message to keep it similar to EMAIL_TOO_LONG

Reply:
Okay, I wasn't sure about that. I was wondering how a user could write a shorter email...